src/api/api.js

```diff
@@ -1,5 +1,5 @@
 import axios from "axios";
 
 export default axios.create({
-    baseURL: "http://localhost:3004/",
+    baseURL: "http://localhost:3007/",
```

Comment:
Why did you change Port number @Yeshwanthnaidu ?

Reply:
Sorry it's a mistake. I'll raise PR again

Reply:
Ok